lib/MetaCPAN/Model/Search.pm

```diff
@@ -44,6 +44,16 @@ sub search_simple {
     return $results;
 }
 
+sub search_first {
```

Comment:
Why is this called `search_first`?  Can we give it a more descriptive name or add a comment?